chapters.md

```diff
@@ -60,13 +113,81 @@ Track/TrackTranslate               |   -   |  X
 
 These other `Elements` belong to the Matroska DVD menu system and are only used when the `ChapProcessCodecID Element` is set to 1.
 
-##### Ordered-Edition and Matroska Segment-Linking
+#### Ordered-Edition and Matroska Segment-Linking
 
-- Hard Linking: `Ordered-Chapters` supersedes the `Hard Linking`.
+- Hard Linking: `Ordered Chapters` supersedes the `Hard Linking`.
 - Soft Linking: In this complex system `Ordered Chapters` are REQUIRED and a `Chapter CODEC` MUST interpret the `ChapProcess` of all chapters.
 - Medium Linking: `Ordered Chapters` are used in a normal way and can be combined with the `ChapterSegmentUID` element which establishes a link to another Matroska file/Segment.
 
-See [the section on the `Linked Segments`](#linked-segments)) for more information about `Hard Linking`, `Soft Linking` and `Medium Linking`.
+See [the section on the `Linked Segments`](notes.md/#linked-segments) for more information about `Hard Linking`, `Soft Linking` and `Medium Linking`.
+
+## ChapterAtom
+The `ChapterAtom` is also called a `Chapter`.
+A `Chapter` element can be used recursively. Such a child `Chapter` is called `Nested Chapter`. 
+
+### ChapterUID
+
+The `ChapterUID` is a mandatory Matroska element with a unique ID with a range from 1 to 18446744073709551615.
+
+### ChapterStringUID
+A unique string ID to identify the `Chapter`.  
+Use for [WebVTT cue identifier storage](http://dev.w3.org/html5/webvtt/#webvtt-cue-identifier).
```

Comment:
You may split the reference in two. The proper RFC citation of the WebVTT spec is `[@!W3C.CR-webvtt1-20190404]`

(we should also do it in the `ChapterStringUID` definition)